### ISSUE TITLE
Feature request: Ansible 2 doesn't show what a task does without "name:". Can we have an option to get the old behavior back (like in Ansible 1.8/1.9)?

### ISSUE THREAD

**Issue by mrosic** [ansible/ansible#14307]
In Ansible 1.x a line like this:
- lineinfile: dest=/root/.bashrc regexp="^export VISUAL=" line="export VISUAL='vim'"

Gives the following output when I execute the playbook:
TASK: [lineinfile dest=/root/.bashrc regexp="^export VISUAL=" line="export VISUAL='vim'"] **\* 
ok: [xy]

However, with Ansible 2 this got changed. The exact same line gives the following output when the playbook is executed:
TASK [lineinfile] **************************************************************
ok: [xy]

Apparently this is the intended behavior for security reasons in order to avoid stuff like passwords being printed on screen. However, it would be nice to have an option to get the behavior from Ansible 1.8/1.9 back (where "name:" was optional).

The reason for this is that when you don't use "name:" all the time some yaml files can be shorter and easier to read, especially if there's suff like 20 "lineinfile" statements in a row. Since there are no unauthorized people watching at my screen I do not have any security gain from the 2.0 behavior.

**Comment by hoxu** [comment]
The new default is a really bad idea, and steers people towards writing useless documentation like:

```
- name: copy ntp.conf to /etc/ntp.conf
  copy: src=ntp.conf dest=/etc/ntp.conf
```

For comparison, only idiots write comments like this in code:

```
// Start listening on port 8080
server.startListening(port=8080);
```

So please, please, consider going back to the old way of working. Or at least provide a configuration option to revert back to old behavior.

**Comment by bcoca** [comment]
closed via #14557
